include/swift/AST/ClangModuleLoader.h

```diff
@@ -40,6 +43,66 @@ enum class ClangTypeKind {
   ObjCProtocol,
 };
 
+/// A path for serializing a declaration.
+class StableSerializationPath {
+public:
+  struct ExternalPath {
+    enum ComponentKind {
```

Comment:
Could you add `print`/`dump` methods for these three types? (Or say only for `StableSerializationPath` and `ExternalPath`)